mvpa2/tests/test_clf.py

```diff
@@ -343,6 +344,7 @@ def test_single_class(self, clf):
         ds = ds[ds.sa.targets == 'L0']  #  only 1 label
         assert(ds.sa['targets'].unique == ['L0'])
 
+        print clf
```

Comment:
at the end there should be no prints in the tests

you can use environ variable  MVPA_DEBUG=TEST   to enable printout of args of running sweep_args decorator ;)